### PR TITLE
Onboarding refactor

### PR DESCRIPTION
**Pull request by aofn**

### CODE REVIEW

File: components/onboarding/onboardingContext.js [hunk 1]
```diff
@@ -36,163 +42,143 @@ function useOnboardingProvider() {
...
+                if (onboardingScript[newIndex].route) router.push(onboardingScript[newIndex].route);
             }
-        }
+        });
     };
 
     const processStep = (offset) => {
         if (!tourInstance.isActive()) {
             // just to make sure, if the tour instance is not active, start the tour
             tourInstance.drive();
```
Comment (aofn) [inline components/onboarding/onboardingContext.js:133]: i tink this line is now also obsolete(?)

**Review by andirueckel — APPROVED** [review]
Tested locally, and basically works; not sure I was able to test *all* edge cases, though.

Content-wise, i.e. the onboarding guide/tour and accompanying texts, still need to worked on to be completed and finalized after merge. @robertschnuell

**Review by robertschnuell — APPROVED** [review]
